<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Prueba de Agilidad</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>711</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>300</y>
      <width>16</width>
      <height>281</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>721</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>102</width>
      <height>94</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>102</width>
      <height>94</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>102</width>
      <height>94</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>102</width>
      <height>94</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>102</width>
      <height>94</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>102</width>
      <height>94</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_4">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>40</y>
      <width>102</width>
      <height>94</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>102</width>
      <height>94</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_5">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>40</y>
      <width>102</width>
      <height>94</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>102</width>
      <height>94</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_6">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>102</width>
      <height>94</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>102</width>
      <height>94</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_7">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>140</y>
      <width>102</width>
      <height>94</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>102</width>
      <height>94</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_8">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>140</y>
      <width>102</width>
      <height>94</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>102</width>
      <height>94</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_9">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>140</y>
      <width>102</width>
      <height>94</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>102</width>
      <height>94</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_10">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>140</y>
      <width>102</width>
      <height>94</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>102</width>
      <height>94</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="_lbl_finPrueba">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>240</y>
      <width>211</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>410</y>
      <width>401</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Informaci√≥n de la prueba</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>301</width>
       <height>121</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Hora inicio de la prueba</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTimeEdit" name="hora_inicio">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Final de la prueba</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTimeEdit" name="hora_final">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Aciertos</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="_lbl_aciertos">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_14">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Errores</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="_lbl_errores">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>300</y>
      <width>401</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Paciente</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>381</width>
       <height>81</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>id:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="_lbl_idpnt">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="_lbl_pntnombre">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>edad:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="_lbl_edadpnt">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="_lbl_apellido">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>genero:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="_lbl_genero">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="_lbl_pntnombre_2">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Nombre y apellido</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>281</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Usuario</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>261</width>
       <height>77</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Username:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="_lbl_username">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_16">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Nombre:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="_lbl_usrnombre">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Apellido:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="_lbl_usrapellido">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>440</y>
      <width>201</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="cmd_ingreso">
         <property name="font">
          <font>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Ingreso</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="RegPnt">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Registrar paciente</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cmd_inicio">
         <property name="font">
          <font>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Inicio</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="font">
          <font>
           <family>Ubuntu</family>
           <pointsize>9</pointsize>
           <italic>false</italic>
          </font>
         </property>
         <property name="text">
          <string>Tiempo de prueba (segs)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="_tiempo_prueba">
         <property name="minimum">
          <number>10</number>
         </property>
         <property name="maximum">
          <number>600</number>
         </property>
         <property name="singleStep">
          <number>5</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <zorder>boton_1</zorder>
   <zorder>line</zorder>
   <zorder>line_2</zorder>
   <zorder>line_3</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>boton_2</zorder>
   <zorder>boton_3</zorder>
   <zorder>boton_4</zorder>
   <zorder>boton_5</zorder>
   <zorder>boton_6</zorder>
   <zorder>boton_7</zorder>
   <zorder>boton_8</zorder>
   <zorder>boton_9</zorder>
   <zorder>boton_10</zorder>
   <zorder>line_4</zorder>
   <zorder>_lbl_finPrueba</zorder>
   <zorder>groupBox</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox_3</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>733</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
